examples/image-segmentation.ipynb

```diff
@@ -145,6 +145,48 @@
     "plt.imshow(multi_class_segmenter.mask)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Integration with ipywidgets\n",
+    "\n",
+    "To reduce friction in the labelling workflow we can add a ipywidget that allows to update current_class with a drop-down menu."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipywidgets as widgets\n",
+    "\n",
+    "N_classes = 3\n",
+    "class_selector = widgets.Dropdown(options=list(range(1, N_classes + 1)), description=\"class\")\n",
+    "\n",
+    "def update(change):\n",
+    "    multi_class_segmenter.current_class = class_selector.value\n",
+    "    multi_class_segmenter.erasing = erasing_button.value\n",
+    "\n",
+    "erasing_button = widgets.Checkbox(value=False, description=\"Erasing\")\n",
+    "erasing_button.observe(update, names=\"value\")\n",
+    "\n",
+    "class_selector.observe(change_class, names=\"value\")\n",
```

Comment:
I think I may have posted something with an error in the issue. Sorry about that!

```suggestion
    "class_selector.observe(update, names=\"value\")\n",
```